replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/SyncHelper.java

```diff
@@ -236,6 +228,32 @@ private Filter buildFilter() {
     return finalFilter;
   }
 
+  @Nullable
+  private Date getModifiedAfter() {
+    final ReplicationStatus replicationStatus =
+        history
+            .getReplicationEvents(config.getName())
+            .stream()
+            .filter(s -> s.getStatus().equals(Status.SUCCESS))
```

Comment:
```suggestion
            .map(ReplicationStatus::getStatus)
            .filter(Status.SUCCESS::equals)
```

Reply:
Can't map because I need the ReplicationStatus object after.

Reply:
do we need the success check? if we are part way through a set and we lose connection it will not be successful but we would still have a valid last modified right? Could we instead just filter until we get the first status that has a last metadata modified set?

Reply:
True. I'll remove the check.

Reply:
❓ Is it guaranteed that getReplicationEvents() returns the events on a given order? This code relies on that order to get the first. If that is the case then let's make sure we document in the `getReplicationEvents()` that they must be returned in that specific order. If that is not the case, then should we not sort them accordingly here?

Reply:
Since we condense the results of configuration runs, we should only have at most one status object per configuration.

Reply:
If that is the case then the why do we have a method that may return multiple? Since by design, it shouldn't happen, shouldn't we change that method to simply return one?

Reply:
We could, it's gone through some iteration. We used to have multiple status objects per configuration, but switched over to just a single condensed status. I think we should talk about how we are going to do status' of configurations though. I personally think multiple entries would be more valuable to an admin.

Reply:
Agreed. In addition, it will be much simpler and optimal in a cloud environment to have multiple entries and let the UI aggregate.